frame/merkle-mountain-range/src/tests.rs

```diff
@@ -85,12 +85,15 @@ fn add_blocks(blocks: usize) {
 
 fn leaf_indices_to_block_numbers(
 	ext: &mut sp_io::TestExternalities,
-	leaf_indices: &Vec<LeafIndex>
+	leaf_indices: &Vec<LeafIndex>,
 ) -> Vec<u64> {
-	leaf_indices.iter().map(|l| {
-		let mmr_size = ext.execute_with(|| crate::Pallet::<Test>::mmr_leaves());
-		ext.execute_with(|| crate::Pallet::<Test>::leaf_index_to_parent_block_num(*l, mmr_size))
-	}).collect()
+	leaf_indices
+		.iter()
+		.map(|l| {
+			let mmr_size = ext.execute_with(|| crate::Pallet::<Test>::mmr_leaves());
+			ext.execute_with(|| crate::Pallet::<Test>::leaf_index_to_parent_block_num(*l, mmr_size))
```

Comment:
Go from parent-num to block-num:

```suggestion
			ext.execute_with(|| crate::Pallet::<Test>::leaf_index_to_parent_block_num(*l, mmr_size)).saturating_add(1u32.into())
```